demos/interactive_face_detection_demo/interactive_face_detection.hpp

```diff
@@ -55,80 +48,18 @@ static const char target_device_message_lm[] = "Optional. Target device for Faci
 "The demo will look for a suitable plugin for device specified.";
 
-/// @brief Message for the maximum number of simultaneously processed faces for Age Gender network
-static const char num_batch_ag_message[] = "Optional. Number of maximum simultaneously processed faces for Age/Gender Recognition network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Head Pose network
-static const char num_batch_hp_message[] = "Optional. Number of maximum simultaneously processed faces for Head Pose Estimation network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Emotions network
-static const char num_batch_em_message[] = "Optional. Number of maximum simultaneously processed faces for Emotions Recognition network " \
-"(by default, it is 16)";
-
-/// @brief Message for the maximum number of simultaneously processed faces for Facial Landmarks Estimation network
-static const char num_batch_lm_message[] = "Optional. Number of maximum simultaneously processed faces for Facial Landmarks Estimation network " \
-"(by default, it is 16)";
-
-/// @brief Message for dynamic batching support for AgeGender net
-static const char dyn_batch_ag_message[] = "Optional. Enable dynamic batch size for Age/Gender Recognition network";
-
-/// @brief Message for dynamic batching support for HeadPose net
-static const char dyn_batch_hp_message[] = "Optional. Enable dynamic batch size for Head Pose Estimation network";
-
-/// @brief Message for dynamic batching support for Emotions net
-static const char dyn_batch_em_message[] = "Optional. Enable dynamic batch size for Emotions Recognition network";
-
-/// @brief Message for dynamic batching support for Facial Landmarks Estimation network
-static const char dyn_batch_lm_message[] = "Optional. Enable dynamic batch size for Facial Landmarks Estimation network";
-
-/// @brief Message for performance counters
-static const char performance_counter_message[] = "Optional. Enable per-layer performance report";
-
-/// @brief Message for GPU custom kernels description
-static const char custom_cldnn_message[] = "Required for GPU custom kernels. "\
-"Absolute path to an .xml file with the kernels description.";
-
-/// @brief Message for user library argument
-static const char custom_cpu_library_message[] = "Required for CPU custom layers. " \
-"Absolute path to a shared library with the kernels implementation.";
-
 /// @brief Message for probability threshold argument
 static const char thresh_output_message[] = "Optional. Probability threshold for detections";
 
-/// @brief Message for face enlarge coefficient argument
-static const char bb_enlarge_coef_output_message[] = "Optional. Coefficient to enlarge/reduce the size of the bounding box around the detected face";
```

Comment:
Please keep and implement this option. This may be a parameter to your local copy of ParseSSD

Reply:
Done